### ISSUE TITLE
The left nav collapse/expand handler should have dynamical title when operated in screen reader.

### ISSUE THREAD

**Issue by lvganfei** [carbon-design-system/carbon#243]
## Detailed description

The title of collapse/expand handler(.bx--inline-left-nav-collapse) in left nav always be "collapse", it should have dynamical title when operated in screen reader.
The title should have two state value, if collapsed, the title should be "expand nav pane", if expanded, the title should be "collapse nav pane". Thus non-sighter user can be informed properly when using screen reader.

> Is this issue related to a specific component? 
>```html
><div class="bx--inline-left-nav-collapse" data-inline-left-nav-collapse="">
>  <a class="bx--inline-left-nav-collapse__link" href="javascript:void(0);">
>  <svg class="bx--inline-left-nav-collapse__arrow">
>    <title>collapse</title>
>    ...
>  </svg>
></div>
>```

### Info 
```
carbon version:
"carbon-components": "^7.8.0",
 "carbon-components-react": "^3.7.0",
 "carbon-icons": "^6.0.4"
```

### Screenshot

![image](https://user-images.githubusercontent.com/1996396/28880496-cd5f52de-77d7-11e7-838b-7cbc1aab7bf1.png)

**Comment by hellobrian** [comment]
Makes sense, we will get a fix out for this soon
